docs/source/conf.py

```diff
@@ -14,11 +14,14 @@
 import sys
 sys.path.insert(0, os.path.abspath('../..'))
 
+from optimizely.version import __version__
```

Comment:
This import should happen before the sys.path command

Reply:
Talked offline and agreed to disregard this comment.